reports/executive_summary/README.txt

```diff
@@ -5,13 +5,25 @@ To Deploy
 2. Log in to Telemetry Self-Serve Data Analysis
 3. Click 'Schedule a job'
 4. Edit or create a job with the following parameters:
-    Job Name:              executive-report-v4
+  4a. Weekly:
+    Job Name:              executive-report-weekly
     Code Tarball:          Upload executive-report-v4-0.X.tar.gz
-    Execution Commandline: ./run.sh
+    Execution Commandline: ./run.sh weekly
     Output Directory:      output
     Output Visibility:     Private
-    Schedule Frequency:    Daily
-    Day of Week:           n/a (Sunday)
+    Schedule Frequency:    Weekly
+    Day of Week:           Monday
     Day of Month:          n/a (1)
-    Time of Day (UTC):     2pm
+    Time of Day (UTC):     10am
     Job Timeout (minutes): 120
```

Comment:
I'm not sure with the change to fewer, larger machines in the cluster. I'll check again. Meanwhile, I'll bump this up to 300 for weekly and 600 for monthly just to be safe.